desktop/containerd/index.md

```diff
@@ -32,186 +32,100 @@ images** checkbox.
-  Stopped: 0
- Images: 0
- Server Version: 22.06.0-beta.0-372-gd3bb8227ce.m
- Storage Driver: stargz
-  driver-type: io.containerd.snapshotter.v1
- Logging Driver: json-file
- Cgroup Driver: cgroupfs
- Cgroup Version: 2
- Plugins:
-  Volume: local
-  Network: bridge host ipvlan macvlan null overlay
-  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
- Swarm: inactive
- Runtimes: runc io.containerd.runc.v2
- Default Runtime: runc
- Init Binary: docker-init
- containerd version: 9cd3357b7fd7218e4aec3eae239db1f68a5a6ec6
- runc version: v1.1.4-0-g5fd4c4d
- init version: de40ad0
- Security Options:
-  seccomp
-   Profile: builtin
-  cgroupns
- Kernel Version: 5.10.124-linuxkit
- Operating System: Docker Desktop
- OSType: linux
- Architecture: aarch64
- CPUs: 5
- Total Memory: 7.653GiB
- Name: docker-desktop
- ID: f4d28427-96df-404c-b47b-647fe5138e2a
- Docker Root Dir: /var/lib/docker
- Debug Mode: false
- HTTP Proxy: http.docker.internal:3128
- HTTPS Proxy: http.docker.internal:3128
- No Proxy: hubproxy.docker.internal
- Registry: https://index.docker.io/v1/
- Labels:
- Experimental: false
- Insecure Registries:
-  hubproxy.docker.internal:5000
-  127.0.0.0/8
- Live Restore Enabled: false
-
-```
-
-### Run a simple container
-
-```console
-$ docker run --rm hello-world
-Unable to find image 'hello-world:latest' locally
-7d246653d051: Download complete
-432f982638b3: Download complete
-46331d942d63: Download complete
-7050e35b49f5: Downloading [>                                                  ]       0B/3.208kB
-
-Hello from Docker!
-This message shows that your installation appears to be working correctly.
-...
-
-```
-
-## Run the container
-
-Run the container specifying port settings:
-
-```console
-$ docker run -p 8080:80 -d nginx
-
-Unable to find image 'nginx:latest' locally
-b95a99feebf7: Download complete
-91d5b6827ff7: Download complete
-fc5ec3f147e4: Download complete
-5b1423465504: Download complete
-1cdde8b981f2: Download complete
-6c0b05f215c0: Download complete
-004f1937a10a: Download complete
-fd61d71c75fe: Download complete
-717bf61a04cf: Download complete
-8b6a7e0df8edbe91dfa77716a5e84ca28348f44f545a0d34c70a8987c56e63e0
-
-```
-
-Confirm the Nginx container is running:
-
-```console
-$ docker ps
-CONTAINER ID   IMAGE     COMMAND                  CREATED         STATUS         PORTS                  NAMES
-93b4d60dfd08   nginx     "/docker-entrypoint.…"   3 seconds ago   Up 3 seconds   0.0.0.0:8080->80/tcp   stoic_mccarthy
-```
-
-You can also check from the browser that Nginx is running:
-
-![Containerd setting in Docker Desktop](../images/containerd_feature_nginx.png){:width="750px"}
-
 ## Building multi-platform images
 
-Sample Dockerfile:
-
-```dockerfile
-# syntax=docker/dockerfile:1
-FROM alpine
+A multi-platform image is a single container image that can run on multiple different architectures.
+Out of the box, the default builder for Docker Desktop doesn't support multi-platform builds.
```

Comment:
Probably not for this PR, but we need to find the right wording for multi-arch / multi-platform images. "a single container image" is somewhat misleading (as it isn't). 

Perhaps we should think about describing it more like a "bundle" of images with multiple variants (for different platforms). Also see some of the UX mockups we're exploring for this; https://github.com/moby/moby/issues/44582#issuecomment-1456938816

- by default, the image can be "used" as if it's a single image
- but don't _fully_ hide the fact that it is a package/bundle that has multiple images included
- and you can "shallow" pull them (only pull the "wrapper" and the variant that matches your platform

Reply:
Yes, agree, we need to come up with a way to better conceptualize this. I'm making a slight change to the wording, but let's continue in follow-up(s)